bugshaker/src/main/java/com/github/stkent/bugshaker/ActivityReferenceManager.java

```diff
@@ -16,24 +16,24 @@
  */
 package com.github.stkent.bugshaker;
 
+import java.lang.ref.WeakReference;
+
 import android.app.Activity;
 import android.os.Build;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
-import java.lang.ref.WeakReference;
-
 public final class ActivityReferenceManager {
 
     @Nullable
-    private WeakReference<Activity> wActivity;
+    private static WeakReference<Activity> wActivity;
```

Comment:
We shouldn't convert this to static, since we are holding on to this state in the class ActivityReferenceManager. Depending upon the value of wActivity, the methods in this class will return different values.